client/blocks/nps-survey/recommendation-select.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { range } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import RecommendationOption from './recommendation-option';
+
+class RecommendationSelect extends PureComponent {
+	static propTypes = {
+		disabled: PropTypes.bool
+	}
+
+	renderOption( value ) {
+		return (
+			<RecommendationOption
+				key={ value }
+				value={ value }
+				selected={ this.props.value === value }
+				disabled={ this.props.disabled }
+				onChange={ this.props.onChange }
+			/>
+		);
+	}
+
+	render() {
+		const values = range( 0, 11 );
```

Comment:
![these-numbers-go-to-11](https://cloud.githubusercontent.com/assets/1587282/24164706/ed99331e-0e44-11e7-9d7a-cc945d4e7c48.gif)

:)